src/protocol/fluvio-protocol-api/src/api.rs

```diff
@@ -9,7 +8,6 @@ use std::fmt::Debug;
 use std::fmt;
 use std::convert::TryFrom;
 
-use log::trace;
 use log::debug;
 
```

Comment:
this need version bump as well since behavior is changed.  not sure how to keep compatible with previous version.  Idea?

Reply:
I made the bump. Not sure, now the encoding-decoding is symmetric for response and request in any case I think that it is a breaking change.

Reply:
Ok.  Based on [trick](https://github.com/dtolnay/semver-trick), let's try this:
* bump up `fluivo-protocol` minor version.
* add fluvio-protocol-api v0.3.0 as dependency. 

in the `fluvio-protocol/src/lib`: change:

```
#[cfg(feature = "api")]
pub mod api {
    pub use fluvio_protocol_api::*;
}

```

to 
```
#[cfg(feature = "api")]
pub mod api {
    pub use fluvio_protocol_api_v2::*;
}
#[cfg(feature = "api")]
pub mod api_v2 {
    pub use fluvio_protocol_api::*;
}
```

where in the `cargo.toml`
```
...
fluvio-protocol-api = { version = "0.3.0", path = "fluvio-protocol-api", optional = true }
fluvio-protocol-api_v2 = { version = "0.2.0",  optional = true, package = "fluvio-protocol-api }
```

And update fluvio_socket to use v2